data-raw/wales-critical-general-acute-beds.R

```diff
@@ -0,0 +1,64 @@
+# ---- Load ----
```

Comment:
Please style all your files in line with the [tidyverse style guide](https://style.tidyverse.org/) which is the standard we use in the humaniverse. The easiest way to do this is to install the [styler](https://github.com/r-lib/styler) package and let it take care of the hard work for you!

Reply:
Additionally for both datasets exported from this script, given the number of columns, perhaps it is better if `tidyr::pivot_longer()` is called. On reflection I should do the same for the NHS England data set. What do you think?

Reply:
Yep I agree - would be better with pivot_longer(). Will make that change now.

Reply:
Great, thanks. I will do the same for the NHS England data.

Reply:
I used a pivot_longer and pivot_wider command to get the data in the format I wanted - could I have done this all in one command?